### PR TITLE
Show command line in 'gradle quarkusDev' in one line

### PR DESCRIPTION
**Pull request by gastaldi**
Original output:
```gradle
➜  my-gradle3 gradle quarkusDev
Listening for transport dt_socket at address: 5005

> Task :quarkusDev
Starting process: 
/home/ggastald/.sdkman/candidates/java/8.0.232-open/jre/bin/java
-Xdebug
-Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=n
-XX:TieredStopAtLevel=1
-Xverify:none
-Djava.util.logging.manager=org.jboss.logmanager.LogManager
-jar
/tmp/my-gradle3/build/my-gradle3-1.0-SNAPSHOT-dev.jar
Args: 
/home/ggastald/.sdkman/candidates/java/8.0.232-open/jre/bin/java
-Xdebug
-Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=n
-XX:TieredStopAtLevel=1
-Xverify:none
-Djava.util.logging.manager=org.jboss.logmanager.LogManager
-jar
/tmp/my-gradle3/build/my-gradle3-1.0-SNAPSHOT-dev.jar
<=========----> 75% EXECUTING [9s]
> :quarkusDev

```

With this PR: 

```gradle                                                                                                                                                                            ➜  my-gradle3 gradle quarkusDev 
Listening for transport dt_socket at address: 5005

> Task :quarkusDev
Launching JVM with command line: /home/ggastald/.sdkman/candidates/java/8.0.232-open/jre/bin/java -Xdebug -Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=n -XX:TieredStopAtLevel=1 -Xverify:none -Djava.util.logging.manager=org.jboss.logmanager.LogManager -jar /tmp/my-gradle3/build/my-gradle3-1.0-SNAPSHOT-dev.jar

2019-12-10 10:04:55,696 INFO  [io.quarkus] (main) Quarkus 999-SNAPSHOT started in 1.158s. Listening on: http://0.0.0.0:8080
2019-12-10 10:04:55,751 INFO  [io.quarkus] (main) Profile dev activated. Live Coding activated.
2019-12-10 10:04:55,751 INFO  [io.quarkus] (main) Installed features: [cdi, resteasy]
<==<=========----> 75% EXECUTING [17s]
> :quarkusDev
```

### CODE REVIEW

File: devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java [hunk 1]
```diff
@@ -345,10 +347,7 @@ public void startDev() {
             pb.redirectErrorStream(true);
             pb.redirectInput(ProcessBuilder.Redirect.INHERIT);
             pb.directory(getWorkingDir());
-            System.out.println("Starting process: ");
-            pb.command().forEach(System.out::println);
-            System.out.println("Args: ");
-            args.forEach(System.out::println);
+            getLogger().lifecycle("Launching JVM with command line: " + pb.command().stream().collect(joining(" ")));
```
Comment (gsmet) [inline devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java]: Is it normal that we don't have the args anymore?

Comment (gastaldi) [reply devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java]: Yes, it shows the same values as `pb.command()`. See https://github.com/quarkusio/quarkus/pull/6057/files#diff-72fac779acece3ea68a6666ee0060ad5L344

**Review by geoand — COMMENTED** [review]
Looks good to me but I can't test because I'm traveling

**Comment by gsmet** [comment]
Force-pushed a rebase as the button is gone.

**Comment by gastaldi** [comment]
I added a sample output to compare the changes: https://github.com/quarkusio/quarkus/pull/6057#issue-350762408

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks.

**Pull request merged**